Packs/PaloAltoNetworks_IoT3rdParty/Playbooks/Quarantine_Device_in_Cisco_ISE_-_PANW_IoT_3rd_Party_Integration_README.md

```diff
@@ -0,0 +1,33 @@
+Playbook to handle incident triggered from PANW Iot (Zingbox) UI to quarantine a device in ISE.
```

Comment:
This playbook handles a single incident triggered in the PANW IoT (Zingbox) UI by creating a corresponding incident in Cortex XSOAR (of type "PANW IoT 3rd Party SIEM Integration - Vulnerability"), and quarantines the device in Cisco ISE.